src/request/providers/xhr.ts

```diff
@@ -18,6 +18,7 @@ import SubscriptionPool from '../SubscriptionPool';
  */
 export interface XhrRequestOptions extends RequestOptions {
 	blockMainThread?: boolean;
+	includeRequestedWithHeader?: boolean;
```

Comment:
That's a good point Kit, we should definitely be looking to at least have JSDoc (and even better README doc too)